### PR TITLE
Added Movie Finder

### PR DESCRIPTION
**Pull request by Asymtode712**
Closes: #269 

- **Title** :  Movie Finder
- **Name:** : Siddheya Kulkarni
- **Idenitfy yourself:** : GSSoC Contributor

### Describe the add-ons or changes you've made 📃

I have added a Movie Finder through which users can easily find every possible information regarding a particular movie.

### How Has This Been Tested? ⚙️

The code has been tested in my local machine

### Screenshots 📷

movie

### Type of change ☑️

What sort of change have you made:

- [X] New feature (non-breaking change which adds functionality)

### Checklist: ☑️

- [X] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [X] This PR does not contain plagiarized content.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly wherever it was hard to understand.
- [X] My changes generate no new warnings.

### Note to reviewers 📄

Please review and merge this PR if it is good to go

### CODE REVIEW

**Comment by Asymtode712** [comment]
@Avdhesh-Varshney Sorry for closing the old PR as the things got messy there, please go through this PR and merge it if everything is okay

File: Vanilla-JS-Projects/Intermediate/Movie-Finder/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- Clone this repository.
+- Navigate to this project directory
+- Run the `index.html` file. 
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+<!-- add the screenshot of the project (Mandatory) -->
+
+![img](./screenshot.webp)
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Movie-Finder/README.md:38]: ss is not shown in readme file

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by Asymtode712** [comment]
@Avdhesh-Varshney Done the requested change, kindly review

**Review by Avdhesh-Varshney — APPROVED** [review]
@Asymtode712 PR Approved 🎉

**Pull request merged**